html-css/week1/sumankaundinya-portfolio/style.css

```
@@ -0,0 +1,109 @@
+/* General Reset */
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body {
+    font-family: 'Arial', sans-serif;
+    line-height: 1.6;
+    color: #fff;
+    background: url('images/suman2.jpg') no-repeat center top / cover;
+    background-attachment: fixed; /* Keeps the image fixed while scrolling */
+}
+
+/* Navigation Bar */
+nav {
+    background: rgba(0, 0, 0, 0.7);
+    color: #fff;
+    padding: 1rem 0;
+    position: sticky;
+    top: 0;
+    z-index: 1000;
+}
+
+nav ul {
+    list-style: none;
+    display: flex;
+    justify-content: center;
+    gap: 20px;
+}
+
+nav ul li a {
+    color: white;
+    text-decoration: none;
+    font-size: 1rem;
+    padding: 8px 16px;
+    transition: background 0.3s;
+}
+
+nav ul li a:hover {
+    background: #007BFF;
+    border-radius: 5px;
+}
+
+/* Hero Section */
+.hero {
+    height: 100vh;
```

```suggestion
    min-height: 100vh;
```
Change to `min-height` to avoid content being cropped on small screens.

---

Sure Jason!!